scripts/install-windows-prereqs.ps1

```diff
@@ -466,9 +466,12 @@ function Install-DCAPDrivers {
     if($LASTEXITCODE -ne 0) {
         Throw "Failed to install nuget DCAP_Components"
     }
-    & "$PACKAGES_DIRECTORY\nuget.exe" install 'Microsoft.Azure.DCAP.Client' -Source "$TEMP_NUGET_DIR;nuget.org" -OutputDirectory "$OE_NUGET_DIR" -ExcludeVersion
+    & "$PACKAGES_DIRECTORY\nuget.exe" install 'Azure.DCAP.Windows' -Version "0.0.1" -OutputDirectory "$OE_NUGET_DIR" 
     if($LASTEXITCODE -ne 0) {
-        Throw "Failed to install nuget Microsoft.Azure.DCAP.Client"
+        Throw "Failed to install nuget Azure.DCAP.Windows"
+    } else {
+        $destDir = Join-Path $env:SystemRoot "System32"
+        Copy-Item -Force "$OE_NUGET_DIR\Azure.DCAP.Windows.0.0.1\dll\*" $destDir
```

Comment:
Same question about adding to System32 directory.

Reply:
Removed.